R/any_is_na_linter.R

```diff
@@ -1,8 +1,8 @@
-#' Require usage of anyNA over any(is.na(.))
+#' Require usage of `anyNA()` over `any(is.na(.))`
```

Comment:
try to make the style here consistent with `any_duplicated_linter.Rd`, given how similar the linters are. e.g. there you added `anyDuplicated(x)` but not here. Wording below should also be basically the same.

Reply:
Yeah, good catch. I have changed this to be more consistent.